app/views/api/v2/hostgroups/main.json.rabl

```diff
@@ -3,6 +3,6 @@ object @hostgroup
 extends "api/v2/hostgroups/base"
 
 attributes :subnet_id, :subnet_name, :operatingsystem_id, :operatingsystem_name, :domain_id, :domain_name,
-           :environment_id, :environment_name, :compute_profile_id, :compute_profile_name, :ancestry,
```

Comment:
Keep ancestry here and in taxonomies for now, since we don't want to break any existing API clients.  We can remove it in the next major API version.